SuperBuild/External_DCMTK.cmake

```diff
@@ -3,6 +3,13 @@ set(proj DCMTK)
 
 # Set dependency list
 set(${proj}_DEPENDENCIES "zlib")
+if(DCMTK_WITH_OPENSSL)
+  if(NOT Slicer_USE_SYSTEM_${proj})
+    list(APPEND ${proj}_DEPENDENCIES OpenSSL)
+  else()
+    # XXX - Add a test checking if system DCMTK supports OpenSSL
```

Comment:
We should add here something. At least log a warning message here.

Reply:
Thanks for the review :pray: 

I will address the following in a follow-up pull request:

```
$ ack "XXX - Add a test"
External_curl.cmake
10:    # XXX - Add a test checking if system curl support OpenSSL

External_DCMTK.cmake
10:    # XXX - Add a test checking if system DCMTK supports OpenSSL
```